ov/server_hardware_type.go

```diff
@@ -9,15 +9,95 @@ import (
 	"strconv"
 )
 
+type VirtualPort struct {
+	Capabilities []string `json:"capabilities,omitempty"` // "capabilities": {}
+	PortFunction string   `json:"portFunction,omitempty"` // "portFunction": "a"
+	PortNumber   int      `json:"portNumber,omitempty"`   // "portNumber": 2
+}
+type SHTPort struct {
+	Mapping               int           `json:"mapping,omitempty"`               // "mapping": 3
+	MaxSpeedMbps          int           `json:"maxSpeedMbps,omitempty"`          // "maxSpeedMbps":3553
+	maxVFsSupported       int           `json:"maxVFsSupported,omitempty"`       // "maxVFsSupported": 23
+	Number                int           `json:"number,omitempty"`                // "number": 1
+	PhysicalFunctionCount int           `json:"physicalFunctionCount,omitempty"` // "physicalFunctionCount": 4
+	SupportedFcGbps       []string      `json:"supportedFcGbps,omitempty"`       // "supportedFcGbps": ""
+	Type                  string        `json:"type,omitempty"`                  // "type": "Ethernet"
+	VirtualPort           []VirtualPort `json:"virtualPorts,omitempty"`          // "virtualPorts": {}
+}
+type Adapter struct {
+	Capabilities    []string  `json:"capabilities,omitempty"`    // "capabilities": {}
+	DeviceNumber    int       `json:"deviceNumber,omitempty"`    // "deviceNumber": 123
+	DeviceType      string    `json:"deviceType,omitempty"`      // "deviceType":"Ethernet"
+	Location        string    `json:"location,omitempty"`        // "location": "Mezz"
+	MaxVFsSupported int       `json:"maxVFsSupported,omitempty"` // "maxVFsSupported": 256
+	MinVFsIncrement int       `json:"minVFsIncrement,omitempty"` // "minVFsIncrement": 8
+	Model           string    `json:"model,omitempty"`           // "model": ""
+	Ports           []SHTPort `json:"ports,omitempty"`           // "ports": {}
+	Slot            int       `json:"slot,omitempty"`            // "slot": 3
```

Comment:
`StorageCapabilities` is part of this attribute as well. Can you check?

Reply:
I missed it in Adapter struct, added it now